felt/src/lib.rs

```diff
@@ -989,16 +1014,18 @@ mod test {
         }
 
         #[test]
+        #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
         // Property test to check that is_multiple_of(x, y) works. Since we're operating in a prime field, is_multiple_of
         // will always be true
         fn is_multiple_of_doesnt_panic(ref x in "(0|[1-9][0-9]*)", ref y in "(0|[1-9][0-9]*)") {
-            let x = Felt::parse_bytes(x.as_bytes(), 10).unwrap();
-            let y = Felt::parse_bytes(y.as_bytes(), 10).unwrap();
-            prop_assert!(x.is_multiple_of(&y));
+                 let x = Felt::parse_bytes(x.as_bytes(), 10).unwrap();
+                 let y = Felt::parse_bytes(y.as_bytes(), 10).unwrap();
+                 prop_assert!(x.is_multiple_of(&y));
         }
     }
 
     #[test]
+    #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
```

Comment:
This mean that when compiling for wasm32 target, you will use the `wasm_bindgen_test` test runner instead of the regular `test` one.

You can read more about it here:
https://crates.io/crates/wasm-bindgen-test
The core problem is that `#[test]` fork the process, which is not supported under a wasm32 runtime